Misc/NEWS.d/next/Library/2021-08-09-13-17-10.bpo-38956.owWLNv.rst

```diff
@@ -0,0 +1 @@
+:class:`argparse.BooleanOptionalAction`'s default value is now longer printed twice when used with :class:`argparse.ArgumentDefaultsHelpFormatter`.
```

Comment:
@mhils i was just going through the PR and found one typo, `value is now longer ....` it should be `value is no longer ....`

Reply:
Good catch, thanks! 🍰